builds/fccsw-build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+#---Create stampfile to enable our jenkins to purge old builds------------------------------
+touch $WORKSPACE/controlfile
+
+#---Set up environment----------------------------------------------------------------------
+cd $WORKSPACE/fccsw
+source $WORKSPACE/fccsw/init.sh
+
+#---Clean build folder----------------------------------------------------------------------
+make purge
+
+#---Run build-------------------------------------------------------------------------------
+ctest -V -S $WORKSPACE/fcc-spi/builds/fccsw-build.cmake
```

Comment:
Is this completely equivalent to `make; make test`? This command has some errors when I try to run it on the build machine, and it would be nice to keep the CI close to what the developers are running!

Reply:
It should be the same. This is just a wrapper for all the `cmake` configuration which also allow us to send the results to [cdash.cern.ch](cdash.cern.ch).

What are the errors that you see?

Reply:
ok! actually it seems that the only problem was that the cmake script assumes the top level repository name is `fccsw`
https://github.com/HEP-FCC/fcc-spi/pull/22/files#diff-6438281251e4c225d223fb19c8ca9ad2R41
(which wasn't the case for me.

but that's fine for me :)

Reply:
yea... it is true this `cmake` configuration does some assumptions and relies on some environment variables, which may not be pretty obvious for a user/developer. However, I don't think users will clone both repos, `fcc-spi` and `FCCSW`, to build the latter. Also it uses the specified configuration define by `CMakeFiles.txt` and it is used in our internal CI, so IMHO it is a good deal.